FreeRTOS/FreeRTOS-Community-Supported-Demos
# Add demo project for Renesas RH850 F1KM-S4 R7F701649

Add demo project for Renesas RH850 F1KM-S4 

Description
-----------
* Add demo project for Renesas RH850 F1KM-S4 R7F701649.
* Test basic functionality.

Test Steps
-----------
To run the demos on Renesas RH850 F1KM-S4 R7F701649, the following steps are required:
* Toolchain download:
Install IDE and Coding Tool: [e2 studio](https://www.renesas.com/us/en/software-tool/e2studio-information-rh850-family#downloads).
Install C Compiler Package for RH850 Family [CC-RH]: [RH850 Compiler CC-RH](https://www.renesas.com/us/en/software-tool/c-compiler-package-rh850-family#downloads).
* Import the project file in e2 studio.
* Build and debug the project

Test result: All test cases passed.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
